commands.js

```diff
@@ -260,6 +260,140 @@ const operations = {
                 });
         },
         help: 'publish pipeline template'
+    },
+    /* Add tag for pipeline template */
+    tag_pipeline_template: {
+        opts: {
+            name: { required: true, abbr: 'n', help: 'Template name' },
+            namespace: {
+                required: true,
+                abbr: 's',
+                help: 'Template namespace'
+            },
+            tag: { abbr: 't', required: true, help: 'Tag name' },
+            version: { abbr: 'v', required: false, help: 'Tag version' }
+        },
+        exec(opts) {
+            return index
+                .tagPipelineTemplate({
+                    name: opts.name,
+                    namespace: opts.namespace,
+                    tag: opts.tag,
+                    version: opts.version
+                })
+                .then(result => {
+                    console.log(JSON.stringify(result));
+                })
+                .catch(err => {
+                    console.error(err);
+                    process.exit(1);
+                });
+        }
+    },
+    get_version_from_pipeline_template_tag: {
+        opts: {
+            name: { required: true, abbr: 'n', help: 'Template name' },
+            namespace: {
+                required: true,
+                abbr: 's',
+                help: 'Template namespace'
+            },
+            tag: { abbr: 't', required: true, help: 'Tag name' }
+        },
+        exec(opts) {
+            return index
+                .getVersionFromPipelineTemplateTag({
+                    name: opts.name,
+                    namespace: opts.namespace,
+                    tag: opts.tag
+                })
+                .then(result => {
+                    console.log(`${result}`);
+                })
+                .catch(err => {
+                    console.error(err);
+                    process.exit(1);
+                });
+        }
+    },
+    /* Remove tag for pipeline template */
+    remove_pipeline_template: {
+        opts: {
+            name: { required: true, abbr: 'n', help: 'Template name' },
+            namespace: {
+                required: true,
+                abbr: 's',
+                help: 'Template namespace'
+            },
+            tag: { abbr: 't', required: true, help: 'Tag name' }
+        },
+        exec(opts) {
+            return index
+                .removePipelineTemplate({
+                    name: opts.name,
+                    namespace: opts.namespace,
+                    tag: opts.tag
+                })
```

Comment:
```suggestion
            }
        },
        exec(opts) {
            return index
                .removePipelineTemplate({
                    name: opts.name,
                    namespace: opts.namespace
                })
```
don't think you need tag for this